lib/player.rb

```diff
@@ -0,0 +1,12 @@
+class Player
+
+	attr_reader :token
+
+	def initialize(token)
+		@token = token
+	end
+
+	def token
+		@token
```

Comment:
With `attr_reader :token`, this method is not needed. `attr_reader` is a macro that automatically adds the getter method to your class for the selected attribute. In this case,
```
def token
  @token
end
```
has already been included in your class through the `attr_reader`.